force-app/main/default/classes/DomainBuilder.cls

```diff
@@ -4,7 +4,7 @@ public abstract class DomainBuilder {
     private static Set<DomainBuilder> objects = new Set<DomainBuilder>();
 
     private Boolean isSetupObject;
-    private Map<SObjectField, DomainBuilder> parentByRelationship = new Map<SObjectField, DomainBuilder>();
+    protected Map<SObjectField, DomainBuilder> parentByRelationship = new Map<SObjectField, DomainBuilder>();
```

Comment:
Why did you do that?

Reply:
So that classes extending DomainBuilder, such as UserStory, has access to this method and can access to their own parents. I think you notice later